imgui-examples/manifest

```diff
@@ -1,11 +1,16 @@
 : 1
 name: imgui-examples
 version: 1.86.0-a.0.z
-summary: imgui C++ library
+summary: Dear ImGui: Bloat-free Graphical User interface for C++ with minimal dependencies
```

Comment:
I would prefer `Executable examples of usage of the Dear ImGui library`

Reply:
Thanks, done.